.gitignore

```diff
@@ -90,3 +90,5 @@ pythonenv*/
 mesa/visualization/templates/external/
 mesa/visualization/templates/js/external/
 
+# Virtual environments
+.venv/
```

Comment:
Looking further, this should be put next to L17 and L18. There is also L87 that they should all be grouped together.

Reply:
Feel free to merge when this has been addressed.